dependency-check-suppressions.xml

```diff
@@ -1,8 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<suppressions
-	xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
```

Comment:
Reset this file once hmcts/ccd-data-store-api#1831 is brought up to match `master` branch.

Reply:
Done in [938cf25](https://github.com/hmcts/ccd-data-store-api/pull/1719/commits/938cf2598f18ab6e7bfe61863665dba6fff24b73)